### ISSUE TITLE
NTR: glutaminylated residue

### ISSUE THREAD

**Issue by srengel** [HUPO-PSI/psi-mod-CV#62]
New Term Request: glutaminylated residue

possible new child term (if appropriate): glutaminylated glutamate

reference: PMID:28801462 Jank T, et al. (2017) Protein glutaminylation is a yeast-specific posttranslational modification of elongation factor 1A. J. Biol. Chem. 292(39):16014-16023

from the intro: "Here we describe, for the first time, a novel type of posttranslational modification, the glutaminylation of a glutamate residue, that occurs within the helix A*–loop–helix A􏰂 region of yeast eEF1A. We show that the glutamine residue is attached via its amino group to the side chain carboxyl group of Glu45 within eEF1A."

**Comment by pmt706** [comment]
MOD:02025 is (I believe) the glutaminylated glutamate that you note (references the same PMID)... although the description doesn't seem correct... thoughts @pabinz or @ricardblum (should we remove the word "free" because it's in the context of a polypeptide)?  We could certainly add in the generic glutaminylated residue following that same rubric, just on the generic amino acid "X".

[Term]
id: MOD:02025
name: 5-glutaminyl glutamic acid
def: "A protein modification that effectively converts the alpha amino group of a glutamine residue to glutaminyl glutamic acid by forming an isopeptide bond with the side chain carboxyl group of a free glutamic acid." [PubMed:28801462]
xref: DiffAvg: "128.13"
xref: DiffFormula: "C 5 H 8 N 2 O 2"
xref: DiffMono: "128.058576"
xref: Formula: "C 10 H 15 N 3 O 5"
xref: MassAvg: "257.24"
xref: MassMono: "257.101167"
xref: Origin: "E"
xref: Source: "natural"
xref: TermSpec: "none"
is_a: MOD:00906 ! modified L-glutamic acid residue

**Comment by ValWood** [comment]
I agree the definition needs fixing. This is the term I requested for 
translation elongation factor EF-1 alpha glutamination so it should refer  to a protein chain
